packages/preset-create-react-app/src/helpers/processCraConfig.ts

```diff
@@ -61,10 +61,8 @@ export const processCraConfig = (
         ...rules,
         {
           oneOf: oneOf.map((oneOfRule: RuleSetRule): RuleSetRule => {
-            if (
-              isString(oneOfRule.loader) &&
-              /[/\\]file-loader[/\\]/.test(oneOfRule.loader)
-            ) {
+            // @ts-expect-error - This is a conflict with types bundled in Storybook.
+            if (oneOfRule.type === 'asset/resource') {
```

Comment:
In theory, we could look for both the old and new loader/type here and keep compat with the previous CRA releases, but not sure it's worth it as the preset has been stable and I can't see us adding back-compat features.